Scatterplots/Scatterplots_360days.Rmd

```diff
@@ -0,0 +1,221 @@
+---
+title: "Cumulative Indicator Scatterplots: 360-Day After First Case"
+output: pdf_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r, echo = FALSE, warning=FALSE, message=FALSE}
+knitr::opts_chunk$set(echo = FALSE, warning=FALSE, message=FALSE)
+library(tidyverse)
+library(lubridate)
+
+# reading in files
+
+cases = read.csv("..\\intermediate_data\\jhu_cases_01.26.2021.csv", check.names = FALSE)
+
+deaths = read.csv("..\\intermediate_data\\jhu_deaths_01.26.2021.csv", check.names = FALSE)
+
+population = read.csv("..\\intermediate_data\\world_bank_population.csv", check.names = FALSE)
+
+indicators <- read.csv("..\\intermediate_data\\ghsi_summary.csv", check.names = FALSE)
+
+
+#pivoting dates into rows and grouping by country code and date
+
+cases_country_date <- cases[7:376]
```

Comment:
For things like this it would probably be easier to use nrow or ncol to dynamically get the length in case we add more rows/columns later

Reply:
This was using Corina's old code. I will fix to use the prepped_data files instead.

Reply:
Done.